frontend/src/pages/Chat/components/Room/Room.tsx

```diff
@@ -1,26 +1,35 @@
+import { useLocalStorage } from "usehooks-ts";
 import Initials from "../../../../components/Initials/Initials";
 import { ChatRoomDto } from "../../../../types/room";
+import { isCurrentUserParticipantOne } from "../../../../utils/utils";
 import styles from "./styles.module.scss";
+import { AccountDto } from "../../../../types/account";
 
 type Props = {
   roomData: ChatRoomDto;
   onRoomClick: () => void;
 };
 
 const Room = ({ roomData, onRoomClick }: Props) => {
+  const [accountDetails] = useLocalStorage<AccountDto | null>("account", null);
```

Comment:
Should be passed as context and not by accessing local storage each time. It's an awkward way to do "global state" in the way you're doing it here.